src/librustdoc/html/render/mod.rs

```diff
@@ -2581,6 +2581,7 @@ fn render_call_locations(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item) {
           <span></span>\
           <h5 id=\"{id}\" class=\"section-header\">\
              <a href=\"#{id}\">Examples found in repository</a>\
+             <a class=\"scrape-help\" href=\"https://doc.rust-lang.org/rustdoc/scraped-examples.html\" target=\"_blank\">?</a>\
           </h5>",
         id = id
     );
```

Comment:
This will make the link have the right release channel.

```suggestion
             <a class=\"scrape-help\" href=\"{doc_prefix}/rustdoc/scraped-examples.html\" target=\"_blank\">?</a>\
          </h5>",
        id = id,
        doc_prefix = DOC_RUST_LANG_ORG_CHANNEL,
    );
```